test/Shared/AzureManaged.Tests/GrpcRetryPolicyDefaultsTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FluentAssertions;
+using Grpc.Core;
+using Grpc.Net.Client.Configuration;
+using Xunit;
+
+namespace Microsoft.DurableTask.Tests;
+
+public class GrpcRetryPolicyDefaultsTests
+{
+    [Fact]
+    public void DefaultConstants_ShouldHaveExpectedValues()
+    {
+        // Assert
+        GrpcRetryPolicyDefaults.DefaultMaxAttempts.Should().Be(10);
+        GrpcRetryPolicyDefaults.DefaultInitialBackoffMs.Should().Be(50);
+        GrpcRetryPolicyDefaults.DefaultMaxBackoffMs.Should().Be(250);
+        GrpcRetryPolicyDefaults.DefaultBackoffMultiplier.Should().Be(2);
+    }
+
+    [Fact]
+    public void DefaultServiceConfig_ShouldHaveExpectedConfiguration()
```

Comment:
Honestly I feel these two tests are kind-of overlapped, I think keeping just this one would be enough